python/tvm/relax/frontend/onnx/onnx_frontend.py

```diff
@@ -483,7 +578,7 @@ class Erf(OnnxOpConverter):
     @classmethod
     def _impl_v13(cls, bb, inputs, attr):
         x = inputs[0]
-        sqrt2 = attach_span(relax.op.sqrt(relax.const(2, x.struct_info.dtype)))
+        sqrt2 = relax.const(_np.sqrt(2), x.struct_info.dtype)
```

Comment:
I know this is just changing it back but curious if there's any difference.

Reply:
I ran into an issue with `relax.op.sqrt` where DefaultGPUSchedule tried to bind threads to it but couldnt since its a scalar. Just using a constant instead avoids the issue (and is more efficient since there's no reason to run sqrt at runtime).